packages/docusaurus-theme-classic/src/theme/DocPage/index.js

```diff
@@ -20,10 +21,17 @@ function DocPage(props) {
     permalinkToId[location.pathname] ||
     permalinkToId[location.pathname.replace(/\/$/, '')];
   const metadata = docsMetadata.docs[id] || {};
-  const {sidebar, description} = metadata;
-
+  const {sidebar, description, title, permalink, cover} = metadata;
+  const {
+    siteConfig: {url},
+  } = useDocusaurusContext();
   return (
-    <Layout noFooter description={description}>
+    <Layout
+      noFooter
+      description={description}
+      title={title}
+      image={cover}
+      permalink={`${url}${permalink}`}>
```

Comment:
How about `baseUrl`? I think it has to be part of the full URL.

Reply:
i think permalink alrd include baseurl for docs